integration-tests/utils/aws/cloudWatchGetLogs.ts

```diff
@@ -112,3 +113,11 @@ const waitForEventWithPatterns = async (
 const pause = (delay: number): Promise<unknown> => {
   return new Promise((r) => setTimeout(r, delay))
 }
+
+export const isLogPresent = (
```

Comment:
If you merge the latest `main` branch this function was renamed and the implementation changed slightly.